spring-pulsar/src/test/java/org/springframework/pulsar/core/PulsarAdministrationBadContextTests.java

```diff
@@ -0,0 +1,41 @@
+package org.springframework.pulsar.core;
+
+import static org.assertj.core.api.Assertions.assertThatIllegalStateException;
+import java.util.Map;
+import org.apache.pulsar.client.admin.PulsarAdmin;
+import org.junit.jupiter.api.Test;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+
+public class PulsarAdministrationBadContextTests extends AbstractContainerBaseTests {
```

Comment:
I think we can delete this now since you moved this into the other test

Reply:
Oops, missed that